src/Main.java

```diff
@@ -0,0 +1,87 @@
+import java.io.*;
+import java.util.List;
+
+
+/**
+ * Created by Komatoz on 25.01.2018.
+ */
+public class Main {
+    public static void main(String[] args) throws MyIOException {
+
+        // firs my Exception
+        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+
+        try {
+            System.out.println(reader.readLine());
+        } catch (IOException e) {
+            throw new MyIOException();
+        }
+
+        //second my Exception
+       myException(null);
+
+
+
+
+    }
+
+
+    static void myException(List list){
+
+        if (list == null) {
+            throw new MyExceptionByRTE("to method myException comes bug");
+        }
+
+    }
+
+
+    static void individualTask() {
+        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+
+        System.out.println("Please write a number");
+        try {
+            int x = Integer.parseInt(reader.readLine());
+            System.out.printf("ok, %d is number", x);
+        } catch (IOException e) {
+            e.printStackTrace();
+        } catch (NumberFormatException ex) {
```

Comment:
Please, add a code that tells to the customer that the entered  characters are not  the number.